storage/game_state.json

```diff
@@ -1 +1 @@
-{"5fcdae068117ff0004bf0d34": {"id": 0, "game_state": [{"module_id": 0, "module_complete": true}], "assessment_state": [{"question_id": 0, "attempts": 0, "complete": false}]}, "5fcdb3468117ff0004bf0d35": {"id": 1, "game_state": [{"module_id": 0, "scene": 0, "line_number": 0, "module_complete": false, "assessment_state": [{"question_id": 0, "attempts": 0, "question_complete": false}]}]}}
\ No newline at end of file
+{"5fcdb3468117ff0004bf0d35": {"id": 1, "game_state": [{"module_id": 0, "module_complete": true, "assessment_progress": [{"question_id": 0, "attempts": 4, "question_complete": true}, {"question_id": 1, "attempts": 4, "question_complete": true}, {"question_id": 2, "attempts": 2, "question_complete": false}], "user_id": "5fcdb3468117ff0004bf0d35"}]}, "5fcdae068117ff0004bf0d34": {"id": 1, "game_state": [{"module_id": 0, "module_complete": true, "assessment_progress": [{"question_id": 0, "attempts": 3, "question_complete": true}, {"question_id": 1, "attempts": 6, "question_complete": true}, {"question_id": 2, "attempts": 0, "question_complete": false}]}]}}
```

Comment:
this is someone's unique id i presume? meaning I don't think we should be putting this in the repo